### PR TITLE
Release 0.0.11

### PR DESCRIPTION
**Pull request by yannikschaelte**
* Rewrite AmiciObjective and PetabAmiciObjective simulation routine to directly use
  amici.petab_objective routines (#209, #219, #225).
* Implement petab test suite checks (#228).
* Various error fixes, in particular regarding PEtab and visualization.
* Improve trace structure.
* Fix conversion between fval and chi2, fix FIM (all #223).

### CODE REVIEW

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pypesto/visualize/parameters.py  1
- test/test_petab_suite.py  8
         

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/amici_objective.py  -1
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5194631&bid=17036955)

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ install:
   - mkdir -p ~/bin/ && ln -s /usr/bin/swig3.0 ~/bin/swig && export PATH=~/bin/:$PATH
   - pip3 install --upgrade -r ./.travis_pip_reqs.txt
   - pip3 install .
+  - pip3 install https://github.com/petab-dev/petab/archive/develop.zip # REMOVE ME
```
Comment (dweindl) [inline .travis.yml]: Remove me ;)

Comment (yannikschaelte) [reply .travis.yml]: nah. one things are more stable again, yes. until then gonna keep.

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pypesto/visualize/parameters.py  1
- test/test_petab_suite.py  8
- test/test_history.py  10
- pypesto/objective/history.py  10
         

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
+ pypesto/objective/amici_objective.py  -1
+ pypesto/profile/profile.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5194631&bid=17036955)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- test/test_history.py  10
- test/test_petab_suite.py  8
- pypesto/objective/history.py  10
- pypesto/visualize/parameters.py  1
         

Complexity decreasing per file
==============================
+ pypesto/objective/amici_objective.py  -1
+ pypesto/profile/profile.py  -1
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5194631&bid=17036955)

**Pull request merged**